src/components/StepperVertical/StepperVerticalItem.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div
+    :class="['whitespace-nowrap cursor-pointer text-lg font-thin',
+             'customHoverTrigger focus-visible:font-bold',
+             { 'text-primary-500 focus-visible:outline-primary-500': !darkMode },
+             { 'text-white focus-visible:outline-white': darkMode },
+             { 'font-bold': active }]"
+    tabindex="0"
+    @keydown.enter="selectStep"
+  >
+    <div
+      :class="['rounded-r-full py-1 my-6',
+               'transition-all ease-linear duration-300',
+               { 'bg-primary-500': !darkMode },
+               { 'bg-white': darkMode },
+               { 'text-white': active && !darkMode },
+               { 'text-primary-500': active && darkMode },
+               { 'w-20 customHoverW': !active },
+               { 'w-full delay-100': active }]"
+    >
+      <span
+        :class="['ml-14 inline-block w-4',
+                 { 'text-white': !darkMode },
+                 { 'text-primary-500': darkMode }]"
+      > {{ index }}
+      </span>
+      <span
+        :class="['ml-7 mr-12',
+                 { 'text-white': active && !darkMode },
+                 { 'text-primary-500': active && darkMode }]"
+      >
+        {{ step.pathName }}
+      </span>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'StepperVerticalItem',
+  props: {
+    step: { type: Object, required: true },
```

Comment:
Should we use a validator to enforce the shape of this object?  It looks like `path` is a required property and it needs to be an array?  We need to make this easier to use for consumer (parent) components.

Reply:
I based these on the existing Stepper steps in campaigns that look like this:

> steps: [
>     { path: '/settings', pathName: 'Choose campaign settings' },
>     { path: '/audience', pathName: 'Add target audience' },
>     { path: '/creative', pathName: 'Compose creative' },
>     { path: '/finalize', pathName: 'Finalize campaign' }
>   ]

this here is the step Object of the step Item, the array is in the `StepperVertical` - and it should be an array of objects containing at least `pathName`
so a validator on the `steps` prop of `StepperVertical` sounds good! will do

Reply:
ah that is bad naming on me. that ` selectStep (step) ` is a keyboard event! `.path` leads to the element, and it simulates the click on the element

Reply:
<img width="531" alt="Screen Shot 2022-04-27 at 10 33 19 AM" src="https://user-images.githubusercontent.com/50080618/165542874-9635b026-cc40-4dc2-9407-5076fba12595.png">